ocs_ci/helpers/e2e_helpers.py

```diff
@@ -392,3 +410,335 @@ def validate_mcg_object_expiration(
 
 def validate_mcg_nsfs_feature():
     logger.info("This is not implemented")
+
+
+def validate_noobaa_rebuild_system(self, bucket_factory_session, mcg_obj_session):
```

Comment:
since it is a disruptive process, please consider creating a fixture instead of function. You can refer [to](https://github.com/red-hat-storage/ocs-ci/blob/master/tests/cross_functional/conftest.py#L80)

Reply:
done